packages/calcite-components/src/components/block/block.scss

```diff
@@ -195,7 +218,7 @@ calcite-action-menu {
   @apply my-2;
 
   .header .title .heading {
-    @apply text-color-1;
+    color: var(--calcite-block-header-color, var(--calcite-color-text-1));
```

Comment:
`--calcite-block-header-text-color`

Reply:
This has `text-color-2` and `text-color-1` for `open` state.